goaviatrix/transit_vpc.go

```diff
@@ -70,6 +70,7 @@ type TransitGatewayAdvancedConfig struct {
 	PeerVnetId                        []string
 	BgpHoldTime                       int
 	EnableSummarizeCidrToTgw          bool
+	EnableMultitierTransit            bool
```

Comment:
Do we need to add this to Advanded Config if we can already get this value from GetGateway/ `list_vpcs_summary` ?

Reply:
no need, removed.